public/views/sponsors.html

```diff
@@ -52,14 +52,18 @@
         <div class="container">
             <div class="row">
                 <div class="col-sm-12 my-title">
-                    <h>TITLE</h>
+                    <h>Interested in Becoming a Sponsor?</h>
                 </div>
             </div>
             <div class="row">
                 <div class="col-sm-12 my-text">
-                    <p>
-                        Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
-                    </p>
+                    <h1>In sponsoring us you'll recieve...</h1>
+                    <ul>
+                        <li> Lots of love </li>
+                        <li> Your logo/name featured on website homepage, posters and team shirts</li>
+                        <li> Our eternal gratitude</li>
```

Comment:
Add a point about affecting the lives of your engineers.
Don't say website homepage. Say just our website.